buildscripts/builddrivers.py

```diff
@@ -91,6 +90,28 @@ def clean_or_remove(self, root_dir, work_dir):
                 
             os.chdir(work_dir)  # change back to the working directory
 
+    def get_local_source(self, source_path):
+        """This assumes interactive mode (not testing) and takes care of getting 
+        the user's input to the path of the local source files for the drivers
+        """
+        while True:
+            if source_path is None:
+                source = input('Enter the full path to the source folder: ')
+            else:
+                source = input("Hit ENTER to use '" + source_path + "' or provide another path to the source folder: ")
+                if len(source) == 0:
+                    source = source_path
+
+            valid = True
```

Comment:
what is this for? it's not used anywhere?

Reply:
only used in Interactive mode but not used in Testing mode